tests/dbus/_misc.py

```diff
@@ -65,9 +69,16 @@ class ServiceR(ServiceABC):
     """
     Handle starting and stopping the Rust service.
     """
+    def __init__(self, not_sim=False):
+        super(ServiceR, self).__init__()
```

Comment:
This would be unnecessary if useless abstract base class were discarded as I suggested in other PR.